frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -30,6 +30,8 @@ export const UNASSIGNED_APPLICATIONS_KEY = '#UNASSIGNED_APP#';
 // Prefix our localStorage items to avoid conflicts if another app ever runs on the same domain.
 export const STORAGE_PREFIX = 'bridge';
 
+export const CONFIGMAP_PREFIX = 'console';
```

Comment:
@rottencandy @invincibleJai Can / should we call this `USERSETTINGS_PREFIX` ? :smile:

Reply:
Updated.